includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+class DesignSystemSharedLinks {
+
+	/** @var DesignSystemSharedLinks */
+	private static $instance;
+
+	private function __construct() {
+	}
+
+	public static function getInstance() {
+		if ( !isset( static::$instance ) ) {
+			static::$instance = new DesignSystemSharedLinks();
+		}
+		return static::$instance;
+	}
+
+	public function setHrefs( $hrefs ) {
+		$this->hrefs = $hrefs;
+		return $this;
+	}
+
+	/**
+	 * @param $name string key for href
+	 * @param $lang string two letter language code
+	 * @return string full URL, in case of lang specific URL missing, default one is returned
+	 */
+	public function getHref( $name, $lang ) {
+		return $this->hrefs[ $lang ][ $name ] ?? $this->hrefs[ 'default' ][ $name ];
+	}
+
+	private $hrefs = [
+		'default' => [
+			'fan-communities' => 'http://fandom.wikia.com/explore',
+			'about' => 'http://www.wikia.com/about',
+			'careers' => 'https://careers.wikia.com',
+			'press' => 'http://fandom.wikia.com/press',
+			'contact' => 'http://fandom.wikia.com/contact',
+			'wikia-gives-back' => 'http://www.wikia.com/wikiagivesback',
+			'terms-of-use' => 'http://www.wikia.com/Terms_of_use',
+			'privacy-policy' => 'http://www.wikia.com/Privacy_Policy',
+			'global-sitemap' => 'http://www.wikia.com/Sitemap',
+			'local-sitemap' => '/wiki/Local_Sitemap',
+			'local-sitemap-fandom' => 'http://fandom.wikia.com/local-sitemap',
+			'api' => 'http://api.wikia.com/wiki/Wikia_API_Wiki',
+			'community-central' => 'http://community.wikia.com/wiki/Community_Central',
+			'support' => 'http://community.wikia.com/wiki/Special:Contact',
+			'create-new-wiki' => 'http://www.wikia.com/Special:CreateNewWiki',
+			'fan-contributor' => null,
+			'wam' => 'http://www.wikia.com/WAM',
+			'help' => 'http://community.wikia.com/wiki/Help:Contents',
+			'media-kit' => 'http://www.wikia.com/mediakit',
+			'media-kit-contact' => 'http://www.wikia.com/mediakit/contact',
+			'social-facebook' => 'https://www.facebook.com/wikia',
+			'social-twitter' => 'https://twitter.com/wikia',
+			'social-reddit' => null,
+			'social-youtube' => 'https://www.youtube.com/user/wikia',
+			'social-instagram' => null,
+			'app-store' => 'https://itunes.apple.com/developer/wikia-inc./id422467077',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "app-store" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)